curator/indexlist.py

```diff
@@ -728,13 +728,17 @@ def filter_empty(self, exclude=True):
         """
         Filter indices with a document count of zero
 
+        Indices that are closed are automatically excluded from consideration
+        due to closed indices reporting a document count of zero.
+
         :arg exclude: If `exclude` is `True`, this filter will remove matching
             indices from `indices`. If `exclude` is `False`, then only matching
             indices will be kept in `indices`.
             Default is `True`
         """
         self.loggit.debug('Filtering empty indices')
         self.empty_list_check()
+        self.filter_closed()
```

Comment:
Oooh, this should precede the line above it. 

Say for instance that you only had closed indices before this filter was called. In the present ordering, you would do the `empty_list_check()` first, followed by removing any closed indices (`filter_closed()`). This would result in an empty list condition which could have been caught by `empty_list_check()`.

Please put the `self.filter_closed()` line above the `self.empty_list_check()` line and this code change should be good to go.

Reply:
Ahh, that makes sense! I'll move that line up.